libpod/oci_conmon_linux.go

```diff
@@ -1064,6 +1064,43 @@ func (r *ConmonOCIRuntime) createOCIContainer(ctr *Container, restoreOptions *Co
 		if restoreOptions.TCPEstablished {
 			args = append(args, "--runtime-opt", "--tcp-established")
 		}
+		if restoreOptions.Pod != "" {
+			// If we are restoring into an existing Pod we need to tell
+			// the runtime to change process and mount labels
+			pod, err := ctr.runtime.LookupPod(restoreOptions.Pod)
+			if err != nil {
+				return errors.Wrapf(err, "pod %q cannot be retrieved", restoreOptions.Pod)
+			}
+			infraContainer, err := pod.InfraContainer()
+			if err != nil {
+				return errors.Wrapf(err, "cannot retrieved infra container from pod %q", restoreOptions.Pod)
+			}
+			if err := infraContainer.syncContainer(); err != nil {
```

Comment:
Need to lock the infra container before doing this

Reply:
Actually, do we need to do this at all? The Mount and Process label should not require a sync

Reply:
I simplified this part.